02_configure_host.sh

```diff
@@ -54,6 +54,16 @@ if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABL
     setup_local_registry
 fi
 
+if [[ ! -z "${SQUID_PROXY}" ]]; then
```

Comment:
instead of using a negative (if not empty) let's check if the variable is defined using -n